src/main/java/by/onliner/page/pages/CatalogPageItem.java

```diff
@@ -1,19 +1,22 @@
-package by.onliner.page;
-
+package by.onliner.page.pages;
 
+import by.onliner.page.AbstractPage;
+import by.onliner.page.components.ShoppingCartPopup;
 import by.onliner.util.Waiter;
 import org.openqa.selenium.WebElement;
 import org.openqa.selenium.support.FindBy;
 
-public class CatalogPageItem extends AbstractPage{
+public class CatalogPageItem extends AbstractPage {
```

Comment:
Please use the same naming for all the classes. Other POs are named according to different pattern